### PR TITLE
Spartan Fishing

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
This pull request is the fishing boat and dock for Sparta

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/spart/dock.xml [hunk 1]
```diff
@@ -10,6 +10,9 @@
   <Identity>
     <Civ>spart</Civ>
     <SpecificName>Limēn</SpecificName>
+    <History>
+        While the city of Sparta was not on the coastline, the Spartan empire reached the south-eastern shore of the Peloponnese peninsula and there would certainly be ports along the sea. Plutarch says that the Spartans were engaged in sea trade, that is until the reforms of Lycurgus did away with it. History tell us that the Spartans had a navy. It was never as powerful however, as Athens.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/spart/dock.xml:14]:
```suggestion
        While the city of Sparta was not on the coastline, the Spartan empire reached the south-eastern shore of the Peloponnese peninsula and there would certainly be ports along the sea. Plutarch says that the Spartans were engaged in sea trade, that is until the reforms of Lycurgus did away with it. History tells us that the Spartans had a navy. It was never as powerful, however, as Athens'.
```

File: 0-ad-history-mod-package/simulation/templates/units/spart/ship_fishing.xml [hunk 2]
```diff
@@ -8,6 +8,10 @@
     <Civ>spart</Civ>
     <SpecificName>Ploîon Halieutikón</SpecificName>
     <Icon>units/hele/ship_fishing.png</Icon>
+    <History>
+        Fish were caught on the south-eastern shore of the Peloponnese peninsula and possibly also along the river of Evrótas, that lead to the city Sparta.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/units/spart/ship_fishing.xml:12]:
```suggestion
        Fish were caught on the south-eastern shore of the Peloponnese peninsula and possibly also along the river Evrótas, that lead to the city of Sparta.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is there anything else I need to change before merging?

**Comment by indoptogopt** [comment]
You can merge it.

**Pull request merged**